qemu/tests/multi_disk.py

```diff
@@ -258,7 +258,8 @@ def _get_data_disks():
         matching_images = process.run(disk_check_cmd, ignore_status=True,
                                       shell=True).stdout_text
         for disk in image_stg_blacklist:
-            if not re.search(disk, matching_images):
+            if not re.search(disk, matching_images) and disk in \
+                    indirect_image_blacklist:
```

Comment:
Good catch, it looks good to me.
BTW, I am confused about the definition `indirect_image_blacklist = "/dev/dasda[\d]*"` in the https://github.com/avocado-framework/avocado-vt/blob/d61b099da99bd0b400683e3ab30c12d41fd3255f/virttest/shared/cfg/machines.cfg#L130-L131
I think it should be `indirect_image_blacklist = "/dev/sda[\d]*"`, like this. but anyway I am not familiar with the s390 arch.
Hi @fbq815 
Would you like to help review whether the `indirect_image_blacklist = "/dev/dasda[\d]*"` is right?
Thanks.